script/pr-api.ts

```diff
@@ -0,0 +1,117 @@
+import * as HTTPS from 'https'
+
+export interface IAPIPR {
+  readonly title: string
+  readonly body: string
+  readonly headRefName: string
+  readonly permalink: string
+}
+
+type GraphQLResponse = {
+  readonly data: {
+    readonly repository: {
+      readonly pullRequest: IAPIPR
+    }
+  }
+}
+
+function gitHubRequest(
+  options: HTTPS.RequestOptions,
+  body: Record<string, any>
+): Promise<Record<string, any> | null> {
+  const opts: HTTPS.RequestOptions = {
+    host: 'api.github.com',
+    protocol: 'https:',
+    method: 'POST',
+    headers: {
+      Authorization: `bearer ${process.env.GITHUB_ACCESS_TOKEN}`,
+      'User-Agent': 'what-the-changelog',
```

Comment:
Should this be a little more... Desktopy? :D like what we use for Desktop callsGitHubDesktop/${appProxy.getVersion()}?

Reply:
Maybe? This is what we use today to get the PRs for the changelog 😂

Reply:
lol. k. Then, suppose it doesn't matter much. :D